{"name": "Проверяем возникновение ошибки при создании двух одинаковых курьеров", "status": "passed", "attachments": [{"name": "stdout", "source": "f8ba1fdb-3aea-4c8a-815b-46b0d28e6c35-attachment.txt", "type": "text/plain"}], "start": 1739434177546, "stop": 1739434177822, "uuid": "0a091d86-c74d-4ffe-bfca-adcd3b75e983", "historyId": "50ebec093ab23e1f5d9397faba9a286a", "testCaseId": "50ebec093ab23e1f5d9397faba9a286a", "fullName": "tests.test_create_courier.TestCreateCourier#test_create_same_courier", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('generate_courier')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "LAPTOP-UNQET23V"}, {"name": "thread", "value": "29312-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_courier"}]}